web/web.go

```diff
@@ -240,9 +241,16 @@ func MustBuildServerURL(pathPrefix string) string {
 	if err != nil {
 		panic(err)
 	}
-	hostname, err := os.Hostname()
+	hostname, err := getServerHostname()
 	if err != nil {
 		panic(err)
 	}
 	return fmt.Sprintf("http://%s:%s%s", hostname, port, pathPrefix)
 }
+
+func getServerHostname() (string, error) {
+	if *hostname != "" {
+		return *hostname, nil
+	}
+	return os.Hostname()
+}
```

Comment:
I'm not sure if I'm more visual (vi is probably too minimal for me), but I prefer to navigate through code using the keyboard, preferably with shortcuts. IntelliJ is the best tool I know so far regarding code navigation and code completion, but it feels too heavy for Go projects. For Sublime I didn't feel so comfortable, even with the `GoSublime` plugin. Do you use any additional plugins in Sublime?